#1126 - Add code coverage support

miroslavpojer opened the pull request:
- Added maven profile into parent pom.
- Added code-coverage section in README.md file.

Closes #1126

miroslavpojer committed:
#1126 - Add code coverage support
- Added maven profile into parent pom.
- Added code-coverage section in README.md file.

wajda commented on README.md:
````diff
@@ -63,6 +63,15 @@ former indicates in which application version the given database schema was intr
 
+## How to measure code coverage
+```shell
+./mvn verify -Pcode-coverage
````
We strongly prefer conditional profile activation (e.g. via `-D`) over the `-P`.

And in general, I guess this is a generic enough build feature that had better be included into the [AbsaOSS/root-pom](https://github.com/AbsaOSS/root-pom) repository.

miroslavpojer replied:
Refactored in commit 30320b0a59f4c2924653158fd2f8ef6a25d00062.

wajda reviewed (changes requested):
The change is implemented in the _root-pom_
AbsaOSS/root-pom#5

miroslavpojer commented:
> The change is implemented in the _root-pom_ [AbsaOSS/root-pom#5](https://github.com/AbsaOSS/root-pom/pull/5)

I will update this PR when new version of root-pom will be released.
Update:
- Done in commit 30320b0a59f4c2924653158fd2f8ef6a25d00062.

miroslavpojer committed:
#1126 - Add code coverage support
- JaCoCo code coverage report removed as it was implemented in root-pom:1.0.4.
- Update of root-pom version in parent-pom module.

wajda reviewed (approved):
LGTM

wajda committed:
Merge branch 'develop' into feature/1126-add-code-coverage-support
